### PR TITLE
pythonPackages.algebraic-data-types: init at 0.1.1

### PR DESCRIPTION
**Pull request by uri-canva**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by uri-canva** [comment]
Depends on https://github.com/NixOS/nixpkgs/pull/69178.

File: pkgs/development/python-modules/algebraic-data-types/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  checkInputs = [
+    hypothesis
+    mypy
+  ];
+
+  meta = with lib; {
+    description = "Algebraic data types for Python";
+    homepage = https://github.com/jspahrsummers/adt;
+    license = licenses.mit;
+    maintainers = with maintainers; [ uri-canva ];
```
Comment (uri-canva) [inline pkgs/development/python-modules/algebraic-data-types/default.nix]: @jspahrsummers Do you want to be a maintainer? :trollface:

**Review by LnL7 — APPROVED** [review]

**Comment by uri-canva** [comment]
Rebased on merged dependency.

File: pkgs/development/python-modules/algebraic-data-types/default.nix (hunk 1 above)
Comment (jspahrsummers) [reply pkgs/development/python-modules/algebraic-data-types/default.nix]: 😨

**Pull request merged**